### ISSUE TITLE
Documentation CI test is failing on master

### ISSUE THREAD

**Issue by fingolfin** [thofma/Hecke.jl#999]
Errors look like this (note that this comes from a *Nemo* jldoctest)
```
Error: doctest failure in ~/.julia/packages/Nemo/xQuHt/src/flint/fmpz.jl:2083-2089

```jldoctest
julia> euler_phi(ZZ(12480))
3072

julia> euler_phi(12480)
3072
```

Subexpression:

euler_phi(ZZ(12480))

Evaluated output:

ERROR: UndefVarError: ZZ not defined
Stacktrace:
 [1] top-level scope
   @ none:1

Expected output:

3072
```

These doctests were only recently added to Nemo, so I am guessing the failures started with the new Nemo release *sigh*.

What I don't understand is why those doctests are run in the first place? There are many more, e.g. for `clog`.

**Comment by thofma** [comment]
Yes, I saw this. The reason is that we added `Nemo` to the modules list, otherwise some docstring could not be included.

**Comment by thofma** [comment]
Fixed in #1006
